supabase/client.py

```diff
@@ -1,11 +1,12 @@
-from typing import Any, Dict
+from typing import Any, Dict, Union
 
 from postgrest import SyncFilterRequestBuilder, SyncPostgrestClient, SyncRequestBuilder
+from postgrest.constants import DEFAULT_POSTGREST_CLIENT_TIMEOUT
 
 from .lib.auth_client import SupabaseAuthClient
 from .lib.client_options import ClientOptions
 from .lib.storage_client import SupabaseStorageClient
-
+from httpx import Timeout
```

Comment:
Can you move this import to the second group of imports? [PEP8](https://peps.python.org/pep-0008/#imports) recommends grouping imports as
1. stdlib
2. related libraries
3. local library imports
Please make this change in the other files as well

Reply:
Okay would do  that now